python/_eos.cc

```diff
@@ -589,6 +614,18 @@ BOOST_PYTHON_MODULE(_eos)
     class_<Observables>("_Observables")
         .def("__getitem__", &Observables::operator[])
         .def("__iter__", range(&Observables::begin, &Observables::end))
+        .def("insert", &Observables::insert, R"(
+            Insert a new observable to EOS by parsing the input string.
+
+            :param name: The name of the new observable.
+            :type name: eos.QualifiedName
+            :param latex: The latex representation of the new observable.
+            :type latex: std::string
+            :param options: The set of options relevant to this new observable. These options apply to all the observables in the expression.
```

Comment:
Can you emphasize the 'all' here for the documentation, i.e., ``**all**``? Generally, we can use all the markup available in reStructuredText, which is also what we're using for the parameter descriptions.